docusaurus/docs/available-scripts.md

```diff
@@ -22,6 +22,11 @@ Builds the app for production to the `build` folder. It correctly bundles React
 
 The build is minified and the filenames include the hashes. See the [production build](production-build.md) section for more information.
 
+React DOM automatically supports profiling in development mode for v16.5+, but since profiling adds some small
+additional overhead it is opt-in for production mode. You can opt-in by using the `--profile` flag. Use `npm run build -- --profile` or `yarn build --profile` to enable profiling in production mode. You can read more about profiling
+using the React.
```

Comment:
I think the period and line break here were the result of a typo

Reply:
oops 😆

Reply:
Ok yeah, that was weird. I must not have saved my changes before pushing 😅